studio-docs/source/schema/schema-reporting-protocol.mdx

```diff
@@ -1,85 +1,155 @@
+## The `reportSchema` mutation
+
+A server that reports its schema to Apollo does so periodically via a GraphQL API hosted at the following endpoint:
 
 ```
 https://schema-reporting.api.apollographql.com/api/graphql
 ```
 
-All requests to this endpoint require a **graph API key** as authentication.
+Specifically, a server executes the `reportSchema` mutation, which has the following shape:
+
+```graphql
+mutation ReportSchemaMutation($coreSchema: String, $report: SchemaReport!) {
+  reportSchema(coreSchema: $coreSchema, report: $report) {
+    inSeconds
+    withCoreSchema
+    ... on ReportSchemaError {
+      code
+      message
+    }
+  }
+}
+```
+
+This mutation always requires a `report` input argument, which has the following _minimum_ shape:
+
+```json
+{
+  "bootID": "abc123",
+  "coreSchemaHash": "F908D0B24486E2B2A032ED019A657E7243BCDB7DC5D939AF69820FE7BADFB200",
+  "graphRef": "docs-example-graph@production"
+}
+```
+
+We'll cover the details of this operation and its inputs in [Protocol sequence](#protocol-sequence).
+
+## Required information
+
+To successfully report a schema to Apollo, your server requires the following information:
+
+### Graph API key
+
+All requests to Apollo's schema reporting endpoint require an `X-API-Key` header:
+
+```yaml
+X-API-Key: service:docs-example-graph:abc123
+```
+
+The value of this header is a **graph API key**. Developers obtain this API key from Apollo Studio and provide it to their server instance on startup.
+
+> We **strongly** recommend that your server accept the graph API key via an environment variable (e.g., `APOLLO_KEY`). This is because API keys are secret credentials that developers should never commit to version control.
 
 <ObtainGraphApiKey />
 
+### Graph ref
+
+Whenever a server reports its schema, it does so to a particular [graph and variant](https://www.apollographql.com/docs/studio/org/graphs/) in Apollo Studio. Apollo uses strings called **graph refs** to describe particular graph-variant combinations:
+
+```
+my-graph-id@my-variant
+```
+
+A developer can obtain a variant's graph ref from the variant's Schema page in Apollo Studio.
+
+As with API keys, we recommend that developers provide their graph ref to your server via an environment variable (e.g., `APOLLO_GRAPH_REF`) that's loaded on startup.
+
+### Boot ID
+
+Whenever an instance of your server boots up, it should generate a UUID to identify itself among _other_ instances of the same server. You provide this ID with each `reportSchema` mutation you execute, so that Apollo can distinguish between requests from different instances.
+
+An instance's boot ID should _not_ persist across restarts.
+
+### Schema hash
+
+Every `reportSchema` mutation requires the SHA-256 hash of the schema string you're reporting to Apollo. You should always **normalize** your schema before generating its hash, as [described below](#schema-normalization).
```

Comment:
We should say how this should be encoded. Should users report this base 16 after taking the SHA-256?